client/client.js

```diff
@@ -0,0 +1,14 @@
+$(function () {
+var socket = io();
+var bruger = "Bob";
```

Comment:
So `bruger` is actually a constant (it never changes). As constatns are usually CAPITALIZED, I would call it `INITIAL_USERNAME`.